controller/DeliveryTool.php

```diff
@@ -182,6 +183,14 @@ public function checkCapacity()
             ->withHeader('Content-Type', 'application/json');
     }
 
+    public function launch1p3(): void
+    {
+        $message = $this->getValidatedLtiMessagePayload();
+
+        LtiService::singleton()->startLti1p3Session($message);
```

Comment:
Don't forget to update `composer.json` with a new version of `extension-tao-lti`, implementing that method.
Feel free to specify the `dev-*` version there so far, just as a placeholder, not to forget bumping the requirement before the merge.